libraries/lib-module-manager/PluginInterface.h

```diff
@@ -80,6 +80,29 @@ using ConfigConstReference =
 
 }
 
+//! Type of plugin registry version information
+using PluginRegistryVersion = wxString;
+
+inline bool Regver_eq(
+   const PluginRegistryVersion &regver1, const PluginRegistryVersion &regver2)
+{
+   return regver1 == regver2;
+}
+
+// Compare registry versions lexicographically as std::wstring
```

Comment:
I was doubtful that wxString comparison might have some strange locale sensitivity?

But no, I decided to be less lazy in my rewrite.  No more std:: strings.  Instead parse the numbers form the string and compare the number sequences lexicographically.  Do it right in case any of the numbers reach double digits.